core/api/src/migrations/000054-lockedString.ts

```diff
@@ -0,0 +1,48 @@
+const tables = [
+  "apiKeys",
+  "apps",
+  "destinations",
+  "groups",
+  "permissions",
+  "profilePropertyRules",
+  "sources",
+  "schedules",
+  "settings",
+  "teams",
+  "teamMembers",
+];
+
+module.exports = {
+  up: async function (migration, DataTypes) {
+    for (const i in tables) {
+      const table = tables[i];
+      await migration.changeColumn(table, "locked", {
+        type: DataTypes.STRING(191),
+        allowNull: true,
+        defaultValue: null,
+      });
+      if (table !== "teams") {
+        await migration.sequelize.query(
+          `UPDATE "${table}" SET locked='config:code' where locked IS NOT NULL`
```

Comment:
@evantahler I think this was a bug. They were `false` so this set everything as 'config:code'
I'd consider switching this to `where locked = true`

Reply:
well shit.

Thanks for finding that!

Reply:
Will be resolved by https://github.com/grouparoo/grouparoo/pull/1048